### PR TITLE
[quant] fix reduce_range warning

### PR DESCRIPTION
**Pull request by terrychenism**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #71027

Summary:
Fix issue #61054. remove warning
reduce_range=True which caused the error message "UserWarning: Please use quant_min and quant_max to specify the range for observers".

Test Plan:
python test/test_quantization.py TestFakeQuantizeOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33484341](https://our.internmc.facebook.com/intern/diff/D33484341)

### CODE REVIEW

**Comment by terrychenism** [comment]
@terrychenism has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33484341).

**Comment by jerryzh168** [comment]
btw, if we write:
```
Fixes: issue link
```
it will close the corresponding issue after this PR is closed

**Comment by jerryzh168** [comment]
can you also try the original example in the issue locally to make sure the warning disappears

**Comment by terrychenism** [comment]
@terrychenism has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33484341).

**Comment by terrychenism** [comment]
> can you also try the original example in the issue locally to make sure the warning disappears

Yes, original example works well

File: test/quantization/core/test_workflow_ops.py [hunk 1]
```diff
@@ -557,7 +557,7 @@ def test_fixed_qparams_fq_module(self, device, X):
     def test_fq_serializable_per_tensor(self):
         observer = default_observer
         quant_min = 0
-        quant_max = 255
+        quant_max = 127
```
Comment (jerryzh168) [inline test/quantization/core/test_workflow_ops.py:560]: wondering why do we change this?

**Review by jerryzh168 — APPROVED** [review]

**Comment by terrychenism** [comment]
@terrychenism has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33484341).

File: test/quantization/core/test_workflow_ops.py (hunk 1 above)
Comment (terrychenism) [reply test/quantization/core/test_workflow_ops.py:560]: the range of default observer is [0, 127] https://github.com/pytorch/pytorch/blob/fbdce8e6caf2de87874e0bc37c12811325849e11/torch/ao/quantization/observer.py#L1415
there's a test error if set quant_max = 255

Comment (jerryzh168) [reply test/quantization/core/test_workflow_ops.py:560]: I see, looks like we have quant_min/quant_max for both FakeQuantize module and Observer module, wonder whether we should dedup them or not

**Pull request closed without merging**